src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -190,13 +197,19 @@ impl<F: TableBuilderFactory> CapacitySplitTableBuilder<F> {
                 }
                 let ret = sstable_store.put_sst(sst_id, meta, data, policy).await;
                 drop(tracker);
+                if let Some(progress_tracker) = task_progress {
+                    progress_tracker.inc_blocks_uploaded();
```

Comment:
Why report blocks uploaded? Meta can not judge when this progress tracker could finish by the number of uploaded blocks.

Reply:
> Meta can not judge when this progress tracker could finish by the number of uploaded blocks. 

Could you clarify what you mean?

We use either increment uploaded blocks or increment in sealed blocks as a measure of progress. In the existing code path, both are required for the entire task to be reported as complete.

Reply:
Oh, I see, I think I made a mistake, this only measures progress at granularity of table, not block.

Reply:
I will change to `inc_tables_uploaded`. Further, I will push progress tracker into each `SstableBuilder`, so that `inc_blocks_built` will be called upon every `build_block`